### ISSUE TITLE
v3.0.x not working on PHP 7.3

### ISSUE THREAD

**Issue by stefandoorn** [akeneo/pim-community-dev#9623]
When doing a `composer install`:

```
> Sensio\Bundle\DistributionBundle\Composer\ScriptHandler::clearCache

In UnitOfWork.php line 2636:
                                                                               
  Warning: "continue" targeting switch is equivalent to "break". Did you mean  
   to use "continue 2"?                                                        
                                                                               

Script Sensio\Bundle\DistributionBundle\Composer\ScriptHandler::clearCache handling the symfony-scripts event terminated with an exception

                                                                                   
  [RuntimeException]                                                               
  An error occurred when executing the "'cache:clear --no-warmup'" command:        
                                                                                   
                                                                                   
                                                                                   
                                                                                   
  In UnitOfWork.php line 2636:                                                     
                                                                                   
    Warning: "continue" targeting switch is equivalent to "break". Did you mean    
     to use "continue 2"?                                                          
                                                                                   

install [--prefer-source] [--prefer-dist] [--dry-run] [--dev] [--no-dev] [--no-custom-installers] [--no-autoloader] [--no-scripts] [--no-progress] [--no-suggest] [-v|vv|vvv|--verbose] [-o|--optimize-autoloader] [-a|--classmap-authoritative] [--apcu-autoloader] [--ignore-platform-reqs] [--] [<packages>]...

The command '/bin/sh -c /usr/local/bin/composer install --no-dev --classmap-authoritative --no-suggest --no-progress' returned a non-zero code: 1
ERROR: Job failed: command terminated with exit code 1
```

Fix is described here and released in `doctrine/orm` version 2.6.3: https://github.com/doctrine/orm/issues/7402. Akeneo requires `doctrine/orm` in version 2.5.14 though.

**Comment by jjanvier** [comment]
Hello @stefandoorn,

The problem is related to some of our Doctrine dependencies that are not compatible with PHP 7.3

This PR https://github.com/akeneo/pim-community-dev/pull/9560 should fix the problem.

**Comment by jmleroux** [comment]
> This PR #9560 should fix the problem.

On master...

**Comment by stefandoorn** [comment]
Great, thanks :-)

**Comment by stefandoorn** [comment]
@jmleroux Will this mean it will land in e.g. v3.1?

**Comment by unreal4u** [comment]
The same happens for the LTS 2.3: will this fix make it to the 2.3 branch?

**Comment by jjanvier** [comment]
@stefandoorn yes, this fix will be land in 3.1 and in the next LTS (3.3)

@unreal4u no, we won't change the requirements of the current 2.3 LTS in a patch.

**Comment by devkbsc** [comment]
I think this is like a loop! in the both issues it's provided a link problem fixed https://github.com/akeneo/pim-community-dev/pull/9560 , 

but i didn't see any solution on the pages
